web/yaamp/core/backend/markets.php

```diff
@@ -902,7 +902,7 @@ function updateNovaMarkets()
 			$sqlFilter = "AND base_coin='{$market->base_coin}'";
 		}
 
-		if (market_get($exchange, $coin->symbol, "disabled", $base)) {
+		if (market_get($exchange, $coin->symbol, "disabled", null, $base)) {
 			$market->disabled = 1;
 			$market->deleted = 1;
```

Comment:
note for later, deleted should not be used... else it can be deleted/recreated in the db. will try to fix that for all at once